stripe/res/values/themes.xml

```diff
@@ -28,10 +28,6 @@
     <style name="StripeGooglePayDefaultTheme" parent="StripePaymentSheetBaseTheme" />
 
     <style name="StripePaymentSheetAddCardTheme">
-        <!-- Required so that keyboard does not cover bottom sheet -->
-        <item name="android:windowSoftInputMode">adjustResize</item>
-        <item name="android:windowIsFloating">false</item>
```

Comment:
I think with this change we can simplify `onUserBack()` to

```
    protected fun onUserBack() {
        keyboardController.hide()
        onBackPressed()
    }
```

https://github.com/stripe/stripe-android/pull/3435/files#diff-c80f2237724f0c4c9d4f26cd2848f0c4100a4179deb02c3c59486d142f538537R105

Reply:
Yes, that works! Done